vfe/vfe.cpp

```diff
@@ -547,7 +550,7 @@ void vfeProcessRenderOptions::ParseError(const char *format, ...)
   va_list marker;
 
   va_start(marker, format);
-  vsnprintf(str, sizeof(str)-2, format, marker);
```

Comment:
Not sure why this is `sizeof(str)-2`, rather than `sizeof(str)-1`; I presume it to be a mistake. If anyone can think of any good reason to leave room for a trailing NUL character _and_ one more byte, let me know.